docs/book/src/getting-started.md

```diff
@@ -1,3 +1,645 @@
+dockermachinetemplate.infrastructure.cluster.x-k8s.io/capi-quickstart-md-0 created
+kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/capi-quickstart-md-0 created
+```
+
+#### Accessing the workload cluster
+
+The cluster will now start provisioning. You can check status with:
+
+```bash
+kubectl get cluster
+```
+
+You can also get an "at glance" view of the cluster and its resources by running:
+
+```bash
+clusterctl describe cluster capi-quickstart
+```
+
+To verify the first control plane is up:
+
+```bash
+kubectl get kubeadmcontrolplane
+```
+
+You should see an output is similar to this:
+
+```bash
+NAME                    CLUSTER           INITIALIZED   API SERVER AVAILABLE   REPLICAS   READY   UPDATED   UNAVAILABLE   AGE    VERSION
+capi-quickstart-g2trk   capi-quickstart   true                                 3                  3         3             4m7s   v1.26.0
+```
+
+<aside class="note warning">
+
+<h1> Warning </h1>
+
+The control plane won't be `Ready` until we install a CNI in the next step.
+
+</aside>
+
+After the first control plane node is up and running, we can retrieve the [workload cluster] Kubeconfig.
+
+```bash
+clusterctl get kubeconfig capi-quickstart > capi-quickstart.kubeconfig
+```
+
+
+### Deploy a CNI solution
+
+Calico is used here as an example.
+
+```bash
+kubectl --kubeconfig=./capi-quickstart.kubeconfig \
+  apply -f https://raw.githubusercontent.com/projectcalico/calico/v3.24.1/manifests/calico.yaml
+```
+
+After a short while, our nodes should be running and in `Ready` state,
+let's check the status using `kubectl get nodes`:
+
+```bash
+kubectl --kubeconfig=./capi-quickstart.kubeconfig get nodes
+```
+
+```bash
+NAME                                          STATUS   ROLES           AGE   VERSION
+capi-quickstart-g2trk-9xrjv                   Ready    control-plane   12m   v1.26.0
+capi-quickstart-g2trk-bmm9v                   Ready    control-plane   11m   v1.26.0
+capi-quickstart-g2trk-hvs9q                   Ready    control-plane   13m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-8tq9v   Ready    <none>          12m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-glnjd   Ready    <none>          12m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-sfzp6   Ready    <none>          12m   v1.26.0
+```
+
+### Clean Up
+
+Delete workload cluster.
+```bash
+kubectl delete cluster capi-quickstart
+```
+<aside class="note warning">
+
+IMPORTANT: In order to ensure a proper cleanup of your infrastructure you must always delete the cluster object. Deleting the entire cluster template with `kubectl delete -f capi-quickstart.yaml` might lead to pending resources to be cleaned up manually.
+</aside>
+
+Delete management cluster
+```bash
+kind delete cluster
+```
+
+## Next steps
+
+See the [clusterctl] documentation for more detail about clusterctl supported actions.
+
+<!-- links -->
+[Experimental Features]: https://cluster-api.sigs.k8s.io/tasks/experimental-features/experimental-features.html
```

Comment:
I think we don't need other providers links here.

Reply:
Yeah I will remove the unnecessary links from here.